packages/integration-tests/projects/suite-web/plugins/index.js

```diff
@@ -81,9 +82,12 @@ module.exports = on => {
             await controller.disconnect();
             return null;
         },
-        startEmu: async () => {
+        startEmu: async version => {
             await controller.connect();
-            const response = await controller.send({ type: 'emulator-start' });
+            const response = await controller.send({
+                type: 'emulator-start',
+                version: version || '2.1.4',
```

Comment:
default version ('2.1.4') should be set in controller, no?

Reply:
done